docs/api/backend-api/material-center.md

```
@@ -327,3 +327,54 @@ tenant 通过前端配置统一 x-lowcode-org 请求头传递，接口中不用
+<p>处理物料同步上传bundle.json文件创建或更新组件</p>
+
+
+### 请求参数
+
+**Headers**
+
+| 参数名称     | 参数值           | 是否必须 | 示例 | 备注 |
+| ------------ | ---------------- | -------- | ---- | ---- |
+| Content-Type | multipart/form-data | 是       |      |      |
+
+**路径参数**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)